docs/source/_topic_map.yml

```diff
@@ -14,8 +14,8 @@ Topics:
         File: installing
       - Name: Quickstart
         File: quickstart
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
+      - Name: Setting up the hypervisor drivers
```

Comment:
> The reasoning behind the rename was: Information wrt Hyper-V virtual switch was expanding. Installation doc did not seem to be the right place for this information. Given that we are already redirecting the users to the Docker Drivers doc section for other drivers, it made sense to add Hyper-V information in there too and provide a link to it from the Installation. 

Sure, but that in itself does not give a reason for the rename. Hyper-V. We are also setting up/configuring the driver.

> However afaiu, please correct me if I am wrong, Hyper-V does not require a Docker machine driver,

It does. All hypervisors do. The driver is the API between docker/lib-machine and the hypervisor.

> only KVM and xhyve need them, so I could not club Hyper-V under the same heading as Docker Machine Drivers. 

You could have

> Based on above discussion it makes sense to give the doc a generic heading like "Setting up the Hypervisor driver plugin" 

"Hypervisor driver plugin" seems wrong to me. If rename, let drop the "docker-machine" part and align with Minikube. Personally I feel we are being less precise though.

> And maybe to be explicit, split the doc into one subheading talking about Hyper-V and another clubbing the KVM and xhyve info under the Docker Machine driver subheading. WDYT?

-1, that would not be needed.